Hazelcast.Net/Hazelcast.Security/DefaultCredentialsFactory.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using Hazelcast.Config;
+using Hazelcast.Util;
+
+namespace Hazelcast.Security
+{
+    public class DefaultCredentialsFactory : ICredentialsFactory
+    {
+        private readonly ICredentials credentials;
+
+        public DefaultCredentialsFactory(ClientSecurityConfig securityConfig, GroupConfig groupConfig)
+        {
+            credentials = InitCredentials(securityConfig, groupConfig);
+        }
+        
+        private ICredentials InitCredentials(ClientSecurityConfig securityConfig, GroupConfig groupConfig)
+        {
+            var credentials = securityConfig.GetCredentials();
+            if (credentials == null) {
+                var credentialsClassname = securityConfig.GetCredentialsClassName();
+                if (credentialsClassname != null) 
+                {
+                    try {
+//                        credentials = ClassLoaderUtil.newInstance(classLoader, credentialsClassname);
```

Comment:
Why is this part commented out ? 
And I could not see a test with only `setCredentialsName` is used. Is there a test for this branch ?